### ISSUE TITLE
Flyout Menu / Accessibility

### ISSUE THREAD

**Issue by tommy-codez** [telekom/scale#401]
Hi there, 
I know its a beta component at the moment. But is it possible to reach the items with keyboard?

I can trigger the menu with return key - but how to reach the items?

best wishes

**Comment by nowseemee** [comment]
Hi @tommy-codez, many thanks for the question. 

The Flyout Menu is currently lacking accessibility features but the good news is that we are working on it. You can expect improvements for the component. I will keep you updated via this thread.

Kind regards,
Daniel

**Comment by tommy-codez** [comment]
Hello again :),

another problem I found is that other triggers (instead of button) 
for example a div or a p-tag nested in the <ScaleMenuFlyout> doesnt get a focus/highlight at all. 

I can use tabindex attribute but the default browser focus doesnt look as good as other focus from other components.

best wishes,
